bookwyrm/templates/book/publisher_info.html

```diff
@@ -18,7 +19,13 @@
 
 <p>
     {% if format and not pages %}
-        {{ format }}
+        {% if format_property and format_property == "AudiobookFormat" and book.audiobook_play_time %}
+            {{ format }}<br />
+            {% trans "Play time:" %}
+            {{ book.audiobook_play_time|localized_duration }}
+        {% else %}
+            {{ format }}
+        {% endif %}
```

Comment:
We should just be checking `book.physical_format` here instead of `format_property` - the latter is `firstof book.physical_format book.physical_format_detail`, and if I understand correctly, `book.physical_format_detail` will never be `AudiobookFormat`, so there's no reason to include it.

I also think the structure of the logic here could be rearranged to match the existing flat style better:
```
{% if book.physical_format == "AudiobookFormat" and book.audiobook_play_time %}
    {{ format }}<br />
    {% trans "Play time:" %}
    {{ book.audiobook_play_time|localized_duration }}
{% elif format and not pages %}
    {{ format }}
{% elif format and pages %}
    {% blocktrans %}{{ format }}, {{ pages }} pages{% endblocktrans %}
{% elif pages %}
    {% blocktrans %}{{ pages }} pages{% endblocktrans %}
{% endif %}
```

I do like nesting to express the conditions better, I just think mixing the two styles is more confusing than sticking with either way. You could also make the whole thing nested, and personally I slightly prefer it, but that's a bigger change:
```
{% if format %}
    {% if book.physical_format == "AudiobookFormat" %}
        {{ format }}
        {% if book.audiobook_play_time %}
            <br />
            {% trans "Play time:" %}
            {{ book.audiobook_play_time|localized_duration }}
        {% endif %}
    {% else %}
      {% if pages %}
          {% blocktrans %}{{ format }}, {{ pages }} pages{% endblocktrans %}
      {% else %}
          {{ format }}
      {% endif %}
    {% endif %}
{% elif pages %}
    {% blocktrans %}{{ pages }} pages{% endblocktrans %}
{% endif %}
```
Honestly, the pages/play time switch adds enough complexity to this it might be worth pulling out as a snippet.